src/components/MainDashboard/OpenPerscription/OpenPerscription.js

```diff
@@ -0,0 +1,86 @@
+import style from "./OpenPerscription.module.css";
+import Accordion from 'react-bootstrap/Accordion';
+
+const OpenPerscription = () => {
```

Comment:
I guess naming the component `OpenPrescription` was not the best choise. We should rename it to `Prescription` only, as it will always show the currently opened prescription. I take the blame for naming it wrong in the design. Please rename the component